# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../sdricoh_cs/no-depmod.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- sdricoh_cs/Makefile.vanilla	2007-04-21 15:35:17.603904250 +0200
+++ sdricoh_cs/Makefile	2007-04-21 15:35:20.876090721 +0200
@@ -10,7 +10,6 @@
 install:
 	if test ! -d $(MDIR) ; then mkdir -p $(MDIR) ; fi
 	install -D -m 644 *.ko $(MDIR)
-	depmod -a
 
 clean:
 	rm -f *.o *.ko *.mod.c .*o.cmd .*o.d .*o.flags Module.symvers
